case-lib/lib.sh

```diff
@@ -364,3 +364,13 @@ logger_disabled()
         return 1
     fi
 }
+
+print_module_params()
+{
+    echo "---------> Printing module parameters <----------"
```

Comment:
`<` is very special in html,  please don't use it in our log, because it may cause problem in web page.